apps/charterafrica/src/payload/blocks/Datasets.js

```diff
@@ -207,6 +207,62 @@ const Datasets = {
                     width: "50%",
                   },
                 },
+                {
+                  name: "openDataset",
```

Comment:
Are all of these extra labels you're adding part of **search and filter** because that's our title.